apps/src/util/sharedVoices.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
```

Comment:
I am a bit confused on how this did get generated here given I don't see any changes to `shared_constants.rb` in this PR.  For my own learning, can you help me understand this?

Reply:
This was a problem with staging that occurred a few days ago. Slack [link](https://codedotorg.slack.com/archives/C0T0PNTM3/p1712940607887079), I'll go ahead and remove the changes